src/chooseGame.ts

```diff
@@ -24,6 +25,8 @@ enum DIRECTION {
   FORWARD = "forward"
 }
 
+const gamesFile = "./tmp/games.json";
```

Comment:
What is this file for?

I see that you write some data there on line 159 but seems like it's not used.

Reply:
Oh, I see that it's for debugging. Ok, let it be that way.